src/components/SearchHit.js

```diff
@@ -78,7 +78,7 @@ export class SearchHit extends Component {
           <SanitizedHtml ruleSet="iiif" htmlString={truncatedHit.after} />
           {' '}
           { truncated && !focused && (
-            <Button className={classes.inlineButton} onClick={showDetails} color="secondary" size="small">
+            <Button className={classes.inlineButton} onClick={showDetails} color="secondary" size="small" aria-describedby={canvasLabel}>
```

Comment:
Should the `aria-describedby` reference an `ID` here?

https://www.w3.org/TR/wai-aria/#aria-describedby

Reply:
😅 alrighty @mejackreed, let me know what you think about this approach. @cbeer suggested I use companion window IDs to deal with the case where there are multiple content search windows open.

Reply:
Looks good!